models/Comments.controller.js

```diff
@@ -0,0 +1,7 @@
+const db = require("../db/connection")
+
+exports.fetchComments = (articleID) =>{
+    return db.query(`SELECT * FROM comments WHERE article_id = ${articleID};`)
+        .then((res)=>{return res.rows})
+        .catch((err)=>{return err})
```

Comment:
We should only have one catch block on any promises chain at the end, our end in this case is in the controller.